uaclient/entitlements/tests/test_fips.py

```diff
@@ -329,26 +331,45 @@ def fake_subp(cmd, *args, **kwargs):
 
         assert 1 == m_remove_apt_config.call_count
 
+    @mock.patch("uaclient.util.is_config_value_true")
+    @mock.patch("uaclient.util.prompt_for_confirmation")
```

Comment:
If there is horizontal space without having to break lines, we can add `, return_value=X` here for these instead of adding more lines to the test below